<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YT Saved Videos</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QWidget" name="centralPanel" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>274</width>
      <height>282</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="tableSortDropDown">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>publish date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>save date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>video title</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>channel title</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="tableSortDirDropDown">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>ascending</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>descending</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tableSearchBar">
       <property name="toolTip">
        <string>Press enter to filter table</string>
       </property>
       <property name="placeholderText">
        <string>Search table</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="videoTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionForceUpdate"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionManageTags"/>
    <addaction name="actionManagePlaylists"/>
   </widget>
   <addaction name="menuImport"/>
   <addaction name="menuManagement"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportPlaylists">
   <property name="text">
    <string>Get playlists</string>
   </property>
  </action>
  <action name="actionManage_tags">
   <property name="text">
    <string>Manage tags</string>
   </property>
  </action>
  <action name="actionManageTags">
   <property name="text">
    <string>Manage tags</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update from playlists</string>
   </property>
  </action>
  <action name="actionForceUpdate">
   <property name="text">
    <string>Force update from playlists</string>
   </property>
  </action>
  <action name="actionManagePlaylists">
   <property name="text">
    <string>Manage playlists</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
